media/js/search-suggestions.js

```diff
@@ -112,7 +80,7 @@
               },
               showarrows: function() {
                 // max-width of $searchFilters is 70% in search-suggestions.styl
-                var max_width = 69 / 100 * $search.width();
+                var max_width = 69 / 100 * $searchForm.width();
```

Comment:
Something is wrong with the math here. I can't trigger it by adding filters. Maybe we want the width of .search-filter-wrapper instead?